configure.ac

```diff
@@ -108,7 +109,8 @@ AS_IF([test x"$enable_debug" != x"no"],
        debug_c_other_flags=$good_flags
        unset good_flags
 
-       CFLAGS="-g -O0 ${base_c_warn_flags} ${debug_c_warn_flags} ${debug_c_other_flags} $CFLAGS"],
+       CFLAGS="-g -O0 ${base_c_warn_flags} ${debug_c_warn_flags} ${debug_c_other_flags} ${CFLAGS_save}"]
+       unset CFLAGS_save,
```

Comment:
Need to move the `]` from the end of the `CFLAGS="-g ...` line to the `unset ...` line (before the `,`).

I.e.

```
       CFLAGS="-g -O0 ${base_c_warn_flags} ${debug_c_warn_flags} ${debug_c_other_flags} ${CFLAGS_save}"
       unset CFLAGS_save],
```

Reply:
Ah, yep, thanks, fixed. I'm surprised that it still built fine for me with that.

Reply:
m4 is a fickle beast.

![my-gosh](https://user-images.githubusercontent.com/617316/51315494-4a56f800-1a20-11e9-8008-2959266dc9db.gif)